src/init.cpp

```diff
@@ -479,14 +479,15 @@ std::string HelpMessage(HelpMessageMode mode)
 
 std::string LicenseInfo()
 {
-    // todo: remove urls from translations on next change
     return CopyrightHolders(strprintf(_("Copyright (C) %i-%i"), 2009, COPYRIGHT_YEAR) + " ") + "\n" +
            "\n" +
            _("This is experimental software.") + "\n" +
            "\n" +
-           _("Distributed under the MIT software license, see the accompanying file COPYING or <http://www.opensource.org/licenses/mit-license.php>.") + "\n" +
+           strprintf(_("The source code is available from %s"), "<https://github.com/bitcoin/bitcoin>") + "\n" +
            "\n" +
-           _("This product includes software developed by the OpenSSL Project for use in the OpenSSL Toolkit <https://www.openssl.org/> and cryptographic software written by Eric Young and UPnP software written by Thomas Bernard.") +
+           strprintf(_("Distributed under the MIT software license, see the accompanying file COPYING or %s"), "<https://opensource.org/licenses/MIT>") + "\n" +
```

Comment:
Translation string freeze is in three days. Let's not invalidate all translations because the URL changed slightly. (The redirect still works fine)

Let's leave this for 0.14.0 or later. Also, the file name (`COPYING`) should be parametrized in the same go.